### ISSUE TITLE
Type of Erased types

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-compiler/Fable#2914]
Right now any type decorated with `Erase` becomes `Any` in Fable AST. This makes sense because originally `Erase` was thought for erased unions representing Typescript unions. However now there are cases where we want to erase the declaration but not necessarily the type. See this comment for example: https://github.com/fable-compiler/Fable/blob/8689186691ab4529c39b445ca85bc1b9fd20ddbb/tests/Main/ImportTests.fs#L113-L124

Another use case is when using a union just to wrap a primitive, then it should probably become the type of the primitive (although I'm not sure if Fable users frequently do this).

**Comment by dbrattli** [comment]
Yes, this would be really nice for Python if it was possible. E.g for the example:

```fs
let fn (a: U2<string, int>) =
    ()
```

It's currently translated to:

```py
def fn(a: Any=None) -> None:
    pass
```

I would like to transpile this to become:

```py
def fn(a: Tuple[str, int]) -> None:
    pass
```
